main.js

```diff
@@ -1,14 +1,28 @@
-const { app, BrowserWindow } = require("electron");
+import { app, BrowserWindow, Tray, Menu, nativeImage } from "electron";
+import { fileURLToPath } from 'url';
+import path, { dirname } from 'path';
+
+let win;
```

Comment:
it's not good name for variable - and I think shortness is  inappropriate- win -> window